assets/v2/stride/assets.json

```diff
@@ -111,5 +111,43 @@
     "decimal": 6,
     "description": "Stride Staked Osmo",
     "image": "stride/stosmo.png"
+  },
+  {
+    "denom": "ibc/D24B4564BCD51D3D02D9987D92571EAC5915676A9BD6D9B0C1D0254CB8A5EA34",
+    "type": "ibc",
+    "base_denom": "uosmo",
+    "base_type": "staking",
+    "dp_denom": "OSMO",
+    "origin_chain": "osmosis",
+    "decimal": 6,
+    "path": "osmosis>stride",
+    "channel": "channel-5",
+    "port": "transfer",
+    "counter_party": {
+        "channel": "channel-326",
+        "port": "transfer",
+        "denom": "uosmo"
+    },
+    "image": "common/osmo.png",
+    "coinGeckoId": "osmosis"
+  },
+  {
+    "denom": "ibc/2475196A6E87EEA47C8878EDE62E99C0FAA14A81B83165E4121E776622EF59BE",
+    "type": "ibc",
+    "base_denom": "basecro",
```

Comment:
@wannabit-yongjoo same here base_denom 
transfer/channel-5/basecro and origin_chain osmosis? eg. https://www.mintscan.io/stride/txs/BF1168D3710BE8CEF89C974F75B7DC5AE7D86C48208508B52B9D3189F00CDD66